### DISCUSSION TITLE
brew install scp no package found

### DISCUSSION THREAD

**Discussion by EdouardMironenkoGitHub** [Homebrew/discussions#2421] [Everyday usage]
Hi, 

So I tried to install on my Mac scp as is instructed on the following page https://pypi.org/project/scp/: 
"pip install scp"

And then I tried to run the example as instructed on the following page https://pypi.org/project/scp/: 

However I seem to be getting an error: 

ModuleNotFoundError: No module named 'paramiko'

I am quite new to this package so if you have any kind of information on how to fix this issue that would be greatly appreciated. It may be that I am doing something incorrectly and I am not sure if I am posting the question in the correct place.

Regards 

Edd

**Comment by gromgit** [comment]
Since you ran `pip install scp`, and not `brew install scp`, this is not a Homebrew issue.

All the examples in the page you pointed to start with:
```
from paramiko import SSHClient
from scp import SCPClient
```
i.e. you need _both_ `paramiko` and `scp` Python modules.

A simple `pip install paramiko` should get things working. If it doesn't, ask the `scp` author for help, or maybe the [Python community on IRC](https://www.python.org/community/irc/). Nothing you've described is Homebrew-specific; everything's about "how to use Python modules".
